bus/bus.c

```diff
@@ -152,8 +152,15 @@ static void gip_bus_remove(struct device *dev)
 		drv->remove(client);
 }
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 15, 0)
-static void gip_bus_remove_compat(struct device *dev)
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(5, 13, 0)
+static int gip_bus_remove_compat_13(struct device *dev)
```

Comment:
There's only gonna be one of these callbacks, so no need to suffix with version. Name them the same is fine.